app/modules/win32ipc/module.cc

```diff
@@ -46,14 +46,15 @@ void CreatePipe(v8::FunctionCallbackInfo<v8::Value> const& args) {
     if ((uintptr_t)(double)pipe.read_end_handle != pipe.read_end_handle ||
         (uintptr_t)(double)pipe.write_end_handle != pipe.write_end_handle) {
         isolate->ThrowException(v8::Exception::Error(
-            v8::String::NewFromUtf8(isolate, "Handle is too large for double")));
+            v8::String::NewFromUtf8(isolate, "Handle is too large for double").ToLocalChecked()));
```

Comment:
this isn't wrong but can use `NewFromUtf8Literal` instead, and would be consistent with the rest of the file that creates string objects from literals.

Reply:
looks like all of the NewFromUtf8Literal calls have to be removed anyways in order to target node 12.13.0 (the version this electron release requires).